packages/components/src/panel/README.md

```diff
@@ -120,6 +120,20 @@ The class that will be added with `components-panel__body`, if the panel is curr
 -   Type: `String`
 -   Required: No
 
+###### disableSmoothScrollIntoView
+
+By default, the `PanelBody` will smooth scroll into view (if needed). Smooth scrolling can be disabled if `disableSmoothScrollIntoView` is set to `true`.
+
+-   Type: `Boolean`
+-   Required: No
+
+###### focusable
+
+By default, the `PanelBody` header is focusable. This can be disabled if `focusable` is set to `false`.
+
+-   Type: `Boolean`
+-   Required: No
+
```

Comment:
Do you think we really need these props as part of the API  or should we just be opinionated and avoid more APIs to support?

Reply:
@youknowriad I think we can be more opinionated. Initially, I wasn't sure how users may respond to the new smooth scrolling behaviour. If it is problematic, we can disable it.

Reply:
What about the focusable prop?

Reply:
I see that the `focusable` prop is passed to `Disclosure` underneath. It probably doesn't work as you're expecting. It just guarantees that, when `true`, the element will remain focusable even when the element is `disabled`. It's the same as our `Button`'s `isFocusable` prop:
https://github.com/WordPress/gutenberg/blob/b98418d1c0c64fa4613c782afce2009fdb0779d5/packages/components/src/button/index.js#L43

If this causes confusion, that's a good signal that we should find a more descriptive name for the prop. Something like `isFocusableWhenDisabled`.

Reply:
Hmm! I'm that case, I'll remove for now :)